templates/gce.tf.erb

```diff
@@ -1,12 +1,6 @@
-provider "google" {
-  region      = "<%= ENV['GCE_REGION'] %>"
-  project     = "govuk-publishing-dns"
-  credentials = "${file("<%= ENV['GCE_CREDENTIALS'] %>")}"
-}
-
 <% records['records'].each_with_index do | record, index | %>
 resource "google_dns_record_set" "<%= record['subdomain'].gsub('.', '_').gsub('@', 'AT') %>_<%= index %>" {
```

Comment:
I think you're right but is that a major problem?

If it is then we can look at e.g. hashing the subdomain and data field values